frontend/assets/stylesheets/objects/_grid.scss

```diff
@@ -113,10 +124,55 @@
     }
 }
 
-/* 3up Stacked (3 > 1)
+/* 3up Stacked (Cols per row: 3 on larger viewport > 1 on smaller)
    ========================================================================== */
 
+// 3 on desktop > 1 on tablet > 1 on mobile
 .grid--3up-stacked {
+    &.grid--equal-heights {
```

Comment:
Yes it would be much better as a standalone modifier. But I did it like this because the breakpoints vary depending on which type of grid class you are using. So a generic one would assume the wrong breakpoint if used in combination with certain grid classes, e.g. if `grid--equal-heights` assumes column view for desktop only, but is used in combination with `grid--3up-stacked-mobile`

I guess a mixin where we could pass in the breakpoint as a variable would solve this. Does that make sense to you or is there some other solution I've overlooked?